tests/src/test/java/alluxio/shell/command/CheckConsistencyCommandTest.java

```diff
@@ -54,5 +57,14 @@ public void inconsistent() throws Exception {
     expected.append("/testRoot/testDir\n");
     expected.append("/testRoot/testDir/testFileB\n");
     Assert.assertEquals(expected.toString(), mOutput.toString());
+
+    mOutput.reset();
+    mFsShell.run("checkConsistency", "-r", "/testRoot");
+    String res = mOutput.toString();
+    Assert.assertTrue(res.contains("/testRoot" + " have: " + "2 files inconsistent.\n")
+        && res.contains("repairing path: " + "/testRoot/testDir\n")
+        && res.contains("repairing path: " + "/testRoot/testDir/testFileB\n"));
+    Assert.assertTrue(!mFileSystem.exists(new AlluxioURI("/testRoot/testDir")));
+    Assert.assertTrue(!mFileSystem.exists(new AlluxioURI("/testRoot/testDir/testFileB")));
```

Comment:
the ufs's /testRoot/testDir and /test/Root/testFileB is delete by ufs, So, the testFileB and testDir need be delete in alluxio.